dump.ps1

```diff
@@ -0,0 +1 @@
+Get-WmiObject -Query 'Select * from Win32_PerfFormattedData_PerfProc_Process' | Select-Object -Property IDProcess,Name,PageFileBytes,PoolNonpagedBytes,PoolPagedBytes,PrivateBytes,VirtualBytes,WorkingSet,CmdLine | ConvertTo-Csv -NoTypeInformation; get-wmiobject -query 'select * From Win32_Process' | Select-Object -Property ProcessId,Name,CommandLine | ConvertTo-Csv -NoTypeInformation
```

Comment:
Is this file required?

Reply:
we're trying to print process information and there is not a straightforward way of doing this in windows